### PR TITLE
Update compare-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
Added DDL to the list.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7258

File: microsoft-365/admin/create-groups/compare-groups.md [hunk 1]
```diff
@@ -35,13 +35,14 @@ In the <a href="https://go.microsoft.com/fwlink/p/?linkid=2052855" target="_blan
 - **Security groups** are used for granting access to resources such as SharePoint sites.
 - **Mail-enabled security groups** are used for granting access to resources such as SharePoint, and emailing notifications to those users.
 - **Shared mailboxes** are used when multiple people need access to the same mailbox, such as a company information or support email address.
+- **Dynamic distribution groups** are  created to expedite the mass sending of email messages and other information within a organization.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/compare-groups.md]:
```suggestion
- **Dynamic distribution groups** are  created to expedite the mass sending of email messages and other information within an organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**